### PR TITLE
API-299 initial blueprint and account settings collection

### PR DESCRIPTION
**Pull request by stuisme**
@brianhyder @moniquejones08 @ejcarr @aoffpo

### CODE REVIEW

File: DudeSettingsApi.PCL.nuspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Auto Generated Unified API SDK</description>
+    <language>en-US</language>
+    <tags>sdk unified api</tags>
+    <dependencies>
+      <group targetFramework="net45">
+		  <dependency id="Microsoft.Bcl" version="1.1.10" />
+		  <dependency id="Microsoft.Bcl.Build" version="1.0.21" />
+		  <dependency id="Microsoft.Net.Http" version="2.2.29" />
+		  <dependency id="Newtonsoft.Json" version="7.0.1" />
```
Comment (brianhyder) [inline DudeSettingsApi.PCL.nuspec]: You will probably want to bump this to 9.0.1

Comment (brianhyder) [inline DudeSettingsApi.PCL.nuspec]: or do a version range

File: DudeSettingsApi.PCL.nuspec [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		  <dependency id="Microsoft.Bcl.Async" version="1.0.168" />
+		  <dependency id="Unirest-APIMATIC" version="1.0.0.2" />
+		  <dependency id="APIMATIC.SDK.Common" version="1.0.0.1" />
```
Comment (brianhyder) [inline DudeSettingsApi.PCL.nuspec]: 1.0.1.27

File: apiary.apib [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+USECONTROLLERPOSTFIX: FALSE
+USEENUMPOSTFIX: FALSE
+USECONSTRUCTORSFORCONFIG: TRUE
+CSHARPDEFAULTNAMESPACE: DSI.SDK.Settings
+JAVADEFAULTNAMESPACE: io.dudesolutions.settings
+BRANDLABEL: Dude Solutions, Inc.
+USERAGENT: Dude SDK
+
+# Dude Settings Api
+![Dude Solutions](https://dudesolutions.com/assets/img/logo.png)
```
Comment (brianhyder) [inline apiary.apib:22]: This logo gives a 404.  @brentpabst Were we removing this or changing it?

File: apiary.apib [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    + page: 1 (number, optional) - {INT} The page to request from the collection.
+    + pagesize: 1000 (number, optional) - {INT} The number of records to load in a page from the collection.
+
+### GET
+
++ Response 200 (application/json)
+    + Attributes (Base Collection Response)
+        + `data` (array[Account Setting], required)
+
+### POST
```
Comment (brianhyder) [inline apiary.apib:94]: We have been creating a separate collection for POST so the endpoint model doesn't include the query parameters.  If we want to change that now that we have the parameters included in a endpoint model we should update the documentation here: https://readme.dudesoln.com/display/EA/Apiary+Documentation+Standards

File: apiary.apib [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+Any response that does not provide a ```200 OK``` response code should be treated as an error.
+
+### GET
+
++ Response 200 (application/json)
+    + Attributes (Heartbeat)
+
+# Group Account Settings
+
+## Account Setting Collection [/accounts/{accountId}/settings{?page,pagesize,orderby}]
```
Comment (brianhyder) [inline apiary.apib:81]: Do you need the `includeAudit` query parameter?

File: apiary.apib (hunk 4 above)
Comment (stuisme) [inline apiary.apib:94]: Done!

File: apiary.apib (hunk 5 above)
Comment (stuisme) [inline apiary.apib:81]: no audit fields in this service

File: DudeSettingsApi.PCL.nuspec (hunk 1 above)
Comment (stuisme) [inline DudeSettingsApi.PCL.nuspec]: done

File: DudeSettingsApi.PCL.nuspec (hunk 2 above)
Comment (stuisme) [inline DudeSettingsApi.PCL.nuspec]: done

**Pull request merged**